test/performance/performance.go

```diff
@@ -223,18 +162,65 @@ func getMetrics(
 		insecure,
 	)
 
+	_, controllerMemAvg := query(
+		thanosHost,
+		token,
+		fmt.Sprintf(
+			"sum(avg_over_time(container_memory_working_set_bytes{container='config-policy-controller'}[%dm:30s])) "+
```

Comment:
Optional: Since this test appears to require running on the hub, it might be nice if we could get propagator stats too.  A future goal would be to simulate multiple managed clusters and policies deployed to them.